src/xdocs/checks/naming/abbreviationaswordinname.xml

```diff
@@ -159,63 +159,70 @@
       </subsection>
 
       <subsection name="Examples" id="Examples">
-        <p>To configure the check:</p>
+        <p id="Example1-config">To configure the check:</p>
         <source>
-&lt;module name=&quot;AbbreviationAsWordInName&quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;AbbreviationAsWordInName&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
         </source>
-        <p>Example:</p>
+        <p id="Example1-code">Example:</p>
         <source>
-public class MyClass extends SuperClass { // OK, camel case
-  int CURRENT_COUNTER; // violation, at most 4 consecutive capital letters allowed
-  static int GLOBAL_COUNTER; // OK, static is ignored
-  final Set&lt;String&gt; stringsFOUND = new HashSet&lt;&gt;(); // OK, final is ignored
+class Example1 extends SuperClass {
+  int CURRENT_COUNTER; // violation 'no more than '4' consecutive capital letters'
```

Comment:
As we do simple file content pasting, why we started to care about correctness such files.
All other PRs had no diff on java code that was easy to review.

Reply:
Looks like only single check is affected by this

Reply:
https://www.diffchecker.com/QgDUdTZ3/, lets preserve OK comments that has extra text like "//ok bla bla bla"

Reply:
hm, we did this Check only movement to Example.java files at https://github.com/checkstyle/checkstyle/pull/13207
so it is actual execution of Checks, good.

it might be that we did it before we agreed to remove "//ok"  and keep "//ok bla".

@stoyanK7 , please restore "//ok bla".

Reply:
> As we do simple file content pasting, why we started to care about correctness such files.

We do not. AbbreviationAsWordInName got valid test files in ea446a7d

Reply:
> it might be that we did it before we agreed to remove "//ok" and keep "//ok bla".

I believe so, commit is at the beginning of the project. Restoring right now.

Reply:
@romani Done.